concourse/scripts/common.bash

```diff
@@ -31,17 +31,26 @@ function os_version() {
 	fi
 }
 
+function os_platform() {
+	if [[ -f "/etc/altlinux-release" ]]; then
+		uname -m
```

Comment:
1. According to the open group, uname is [guaranteed](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/uname.html) to return only hardware name with `-m` option. Processor architecture (`-p`) is a [Linux](https://man7.org/linux/man-pages/man1/uname.1.html)-specific thing.
2. Anyway, it is incorrect to rely on processor architecture as we can possibly use 32-bit OS on 64-bit CPU (though we don't build GPDB for 32-bit, but anyway).

Maybe we should better get rid of `uname -p` and rename all the result names? It seems to be a candidate for the upstream PR as well.

Reply:
I was confused by solaris uname [man page](https://docs.oracle.com/cd/E19109-01/tsolaris8/817-0879/6mgl9vnhr/index.html) that manifests:
>  -m Print the machine hardware name (class). **Use of this option is discouraged; use uname -p instead.** See NOTES section below.
-p Print the **current host's ISA** or processor type.

Though, of course, we may decrease patch size by replacing to `uname -m` because we can test this command on all supported platforms.